example/src/lib.rs

```diff
@@ -37,11 +37,28 @@ enum Vehicle {
     Car { brand: String, color: String },
 }
 
+#[derive(Serialize, TS)]
+struct Point<T>
+where
+    T: TS,
```

Comment:
I would have hoped we didn't need to specify this `where` clause manually, but I felt rewriting the `where` clause was a little bit too much to hope for in this first version...

Reply:
On second thought, if we did automatic rewriting, we wouldn't just need to rewrite it in the struct definition, but also for the `impl` blocks. That seems unreasonable, and I think it would also be very intuitive if you need to do manually for the `impl` blocks and *not* for the struct definition. So I'm starting to think it should stay like this.